environments/includes/selenium.base.yml

```diff
@@ -1,27 +1,8 @@
 version: "3.5"
 services:
-  selenium-hub:
```

Comment:
A few places in docs this term will need to be replaced with simply `selenium` so that users don't get tripped up setting `SELENIUM_HOST` incorrectly.